cmd/root.go

```diff
@@ -0,0 +1,106 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/github/hub/github"
+	"github.com/github/hub/ui"
+	"github.com/github/hub/utils"
+
+	"github.com/rgreinho/keeparelease/keeparelease"
+	"github.com/spf13/cobra"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "keeparelease",
+	Short: "Create beautiful GitHub releases.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// Get the flags.
+		extract, err := cmd.Flags().GetBool("extract")
+		utils.Check(err)
+
+		// Read the last release information from the Changelog.
+		title, content, err := keeparelease.ReadChangelog()
+		if err != nil {
+			return err
+		}
+
+		// If extract only, simply display the content of the last release.
+		if extract {
+			fmt.Print(content)
+			return nil
+		}
+
+		// Inspect the local repo.
+		localRepo, err := github.LocalRepo()
+		utils.Check(err)
+		project, nil := localRepo.MainProject()
+		utils.Check(err)
+
+		// Prepare the Hub client.
+		gh := github.NewClient(project.Host)
+
+		// Prepare the release.
+		tag, err := cmd.Flags().GetString("tag")
```

Comment:
Either check the error or explicitly ignore it by changing this to `tag, _ := ...`.

I'd expect `go lint` to catch this.